featurecontext/featurecontext.go

```diff
@@ -14,6 +14,11 @@ type User struct {
 	User      *userv1beta1.User
 }
 
+type ResourceAlias struct {
+	Ref *providerv1beta1.Reference
+	Info *providerv1beta1.ResourceInfo
```

Comment:
I would prefer to not store information that can change rather quickly (eg. mtime). Doing a stat when actually inspecting the resource shouldn't be that bad?

Reply:
I want to do a stat and compare checksums and etags with the previos values.